src/utils/getOptionalArgument.ts

```diff
@@ -0,0 +1,6 @@
+export default function (key: string, value?: string): string[] {
+  if (value !== undefined) {
+    return [key, value];
+  }
+  return [];
```

Comment:
Sam-ish stuff (add `&& value !== null` to the if statement), but short and sweet
```suggestion
  return value ?? [key, value]  : [];
```